scbw/result.py

```diff
@@ -115,7 +115,7 @@ def _process_files(self) -> None:
             return
 
         winner_score_file = [file for file, score in scores.items() if score.is_winner][0]
-        nth_player = int(winner_score_file.replace("/scores.json", "").split("_")[-1])
+        nth_player = int(winner_score_file.replace("/scores.json", "").replace("\\scores.json", "").split("_")[-1])
```

Comment:
We can use `os.sep` instead

Reply:
And ideally we could do that wherever these os-specific slashes are in the code ...

Reply:
since Windows support both versions, it is usually create more problems to use such platform dependent separators, if not normalize them via API to be consistent with platform findings. What if we have some path created like this `c:\test/game\score_0.json`. I'm all for using `os.sep` but usually it forces to use path functions holistically and don't depend on the string manipulations entirely.

But we could try `os.sep` and looks how it goes.

Reply:
I can merge this for a quick fix, and then code can be updated with calls for `os.sep` and `os.path.join`, how about that?

Reply:
Reasonable, Let me create separate issue to track that. I could not promise that I make this immediately.

Reply:
Add https://github.com/Games-and-Simulations/sc-docker/issues/69 to track this